sdk/textanalytics/Azure.AI.TextAnalytics/tests/TextAnalyticsClientLiveTests.cs

```diff
@@ -57,6 +57,119 @@ public async Task DetectLanguageWithCountryHintTest()
             Assert.AreEqual("Spanish", language.Name);
         }
 
+        [Test]
+        public void DetectLanguageWithErrorCountryHintTest()
+        {
+            TextAnalyticsClient client = GetClient();
+            string input = "Este documento está en español";
+
+            Assert.ThrowsAsync<RequestFailedException>(() => client.DetectLanguageAsync(input, "COLOMBIA"));
```

Comment:
Is there anything more we could assert, it might be worth to assert the exception message, maybe?